migration/identity-migration/src/main/java/io/camunda/migration/identity/transformer/AuthorizationTransformer.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.migration.identity.transformer;
+
+import io.camunda.migration.identity.dto.Role.Permission;
+import io.camunda.service.AuthorizationServices.PatchAuthorizationRequest;
+import io.camunda.zeebe.protocol.record.value.AuthorizationResourceType;
+import io.camunda.zeebe.protocol.record.value.PermissionAction;
+import io.camunda.zeebe.protocol.record.value.PermissionType;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class AuthorizationTransformer {
+
+  static final List<AuthorizationResourceType> RESOURCE_TYPES =
+      Arrays.asList(
+          AuthorizationResourceType.PROCESS_DEFINITION,
+          AuthorizationResourceType.DECISION_DEFINITION,
+          AuthorizationResourceType.DECISION_REQUIREMENTS_DEFINITION);
```

Comment:
:question: Why only these resources?

Reply:
It's been limited to these three because in the referenced google doc, old identity permissions were mapped to these 3 resource types.